abstracts/_colors.scss

```
@@ -34,7 +34,8 @@ $color-palate: (
   ac-rigel: #a29bfe,
   ac-agena: #6c5ce7,
   ac-spica: #fd79a8,
-  ac-mimosa: #e84393
+  ac-mimosa: #e84393,
+  floating-blue: #204167
```

```suggestion
  ac-floating-blue: #204167
```
lets keep up with convention :)

---

fixed